lib/task.js

```diff
@@ -120,7 +120,7 @@ Task.prototype.help = function (callback) {
   }
 
   log.info("help",
-    "\n\n" + chalk.green.bold("Usage") + ": \n\n  builder " + action + " [flags] [task]" +
+    "\n\n" + chalk.green.bold("Usage") + ": \n\n  builder " + action + " <action> <task(s)>" +
```

Comment:
There are many flags and conditionally different. And they can go anywhere.

See: https://github.com/FormidableLabs/builder#builder-actions